html-forms/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>HTML Forms</title>
+    </head>
+    <body>
+        <h1>Coding Form!</h1>
+        <form action="" method="get">
+            <div>
+                <label for="email"> Email: <br></label>
+                    <input type="email" name="email"/> <br>
+            </div>
+            <div>
+                <label for="select">
+                    Why is coding cool? <br>
+                    <select name="whycoding">
+                        <option value="select" selected>Select an option</option>
+                        <option value="interesting">Because it's interesting</option>
+                        <option value="fun">Because it's fun</option>
+                        <option value="powerful">Because it's powerful</option>
+                    </select>
+                </label>  
+            </div>
+            <div>
+                <label for="radio">Favorite Language (choose one)<br></label>
+                <input type="radio" name="favoritelanguage" value="HTML"/> HTML
+                <input type="radio" name="favoritelanguage" value="CSS"/> CSS
+                <input type="radio" name="favoritelanguage" value="JavaScript"/> JavaScript
```

Comment:
The text for each of these inputs should be a `label` element that is properly configured to match with the `input`